<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="20" started-at="2017-03-21T11:09:36Z" finished-at="2017-03-21T11:09:36Z">
    <groups>
    </groups>
    <test name="NX512J" duration-ms="20" started-at="2017-03-21T11:09:36Z" finished-at="2017-03-21T11:09:36Z">
      <class name="com.zzy.appium.test.HangMei">
        <test-method status="FAIL" signature="beforeClass(java.lang.String, java.lang.String)[pri:0, instance:com.zzy.appium.test.HangMei@4b4523f8]" name="beforeClass" is-config="true" duration-ms="0" started-at="2017-03-21T19:09:36Z" finished-at="2017-03-21T19:09:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[NX512J]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[4491]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at java.io.Reader.<init>(Unknown Source)
	at java.io.InputStreamReader.<init>(Unknown Source)
	at com.zzy.appium.utils.ProUtil.readProperties(ProUtil.java:40)
	at com.zzy.appium.utils.ProUtil.<init>(ProUtil.java:24)
	at com.zzy.appium.test.HangMei.beforeClass(HangMei.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="SKIP" signature="afterClass()[pri:0, instance:com.zzy.appium.test.HangMei@4b4523f8]" name="afterClass" is-config="true" duration-ms="0" started-at="2017-03-21T19:09:36Z" finished-at="2017-03-21T19:09:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="SKIP" signature="f()[pri:0, instance:com.zzy.appium.test.HangMei@4b4523f8]" name="f" duration-ms="0" started-at="2017-03-21T19:09:36Z" finished-at="2017-03-21T19:09:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
      </class> <!-- com.zzy.appium.test.HangMei -->
    </test> <!-- NX512J -->
  </suite> <!-- Suite -->
</testng-results>
